.github/actions/trun/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "trun"
+description: "asdf"
+inputs:
+  dir:
+    required: true
+    description: source directory of files to run
+  browser:
+    description: run files in a browser
+  importMap:
+    description: import map
+  concurrency:
+    description: concurrency
+  ignore:
+    description: ignore file
+runs:
+  using: "docker"
+  image: "Dockerfile"
```

Comment:
This setup is only necessary for the browser example workflow, correct? If so, should we forgo the action definition altogether and inline these instructions in the browser example workflow?

Reply:
should be possible

Reply:
Nah i dont think it works. dockerfiles only work in actions not workflows

Reply:
You can specify `container:` on a job – https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container#example-running-a-job-within-a-container